azkaban-solo-server/build.gradle

```diff
@@ -4,6 +4,8 @@ dependencies {
   compile(project(':azkaban-common'))
```

Comment:
I think we can get rid of 'azkaban-common' and the build should still work fine since it's transitively included.
I'm curious how slf4j is being used ?